include/prereq-build.mk

```
@@ -159,7 +159,7 @@ $(eval $(call SetupHostCommand,python,Please install Python >= 3.5, \
 	python3.7 -V 2>&1 | grep 'Python 3', \
 	python3.6 -V 2>&1 | grep 'Python 3', \
 	python3.5 -V 2>&1 | grep 'Python 3', \
-	python3 -V 2>&1 | grep -E 'Python 3\.[5-9]\.?'))
+	python3 -V 2>&1 | grep -E 'Python 3\.([5-9]|10)\.?'))
```

This approach will break in a year when Python 3.11 is released
```suggestion
	python3 -V 2>&1 | grep -E 'Python 3\.([5-9]|[0-9]+)?\.?'))
```